assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -336,6 +336,77 @@ function initialize(api) {
     },
   });
 
+  api.createWidget("assigned-to-first-post", {
+    html(attrs) {
+      const topic = attrs.topic;
+      const [assignedToUser, assignedToGroup, indirectlyAssignedTo] = [
```

Comment:
Small tip: You could destructure this like `{ assignedToUser, assignedToGroup, indirectlyAssignedTo } = topic`;

Reply:
Actually, in this case it will not work as serializer is returning `assigned_to_user`, and we want to have JS style variable `assignedToUser`